docs/aerodromes/classc/EastSale.md

```diff
@@ -4,7 +4,153 @@
+
+## Low Approach
+- By day, pilots of local aircraft may request a low approach.
+- Pilots are responsible for ensuring that no collision risk exists and that there is suitable spacing to continue the approach and for the potential/subsequent go-around.
+
+## Standard Taxi Routes
+| Runway     | Outbound Route                   | Inbound Route                   |
+| ---------- | -------------------------------- | --------------------------------|
+| Runway 27  | E2 or Eastern Check Bays         | D2                              |
+| Runway 09  | D2 or Western Check Bays         | E2                              |
+| Runway 22  | E2 or Eastern Check Bays         | J                               |
+| Runway 04  | J                                | D2                              |
+
+<figure markdown>
+![ESL TAXI INBOUND](img/esl_taxi_inbound.png){ width="700" }
+<figcaption>ESL TAXI INBOUND</figcaption>
+</figure>
+
+<figure markdown>
+![ESL TAXI OUTBOUND](img/esl_taxi_outbound.png){ width="700" }
+<figcaption>ESL TAXI OUTBOUND</figcaption>
+</figure>
+
+## Helicopter Operations
+- **Helicopter Circuits**:  
+  - When RWY09/27 is being used for fixed-wing circuit training, it is preferred that helicopters utilise threshold runway 04 (runway 09/27 direction) for landing and take-off to increase segregation between final approach and upwind segments.
+  - When helicopters require use of TWY A (Pad Alpha) for circuit training, base turns should be sequenced to avoid conflict during the final approach segment.
+
+## Lanes
+The 16 individual training areas within the ESL military airspace are separated by 4 outbound lanes.
+These lanes are used by aircraft transiting to/from exterior training areas or for entry and exit of ESL military airspace.
+Lanes are defined by GNSS waypoints situated at 12, 35, and 50 NM from YMES AD.
+
+### Outbound Lanes
+
+| Lane Direction | 12NM GNSS Waypoint | 35NM GNSS Waypoint | 50NM GNSS Waypoint | Bearing |
+| -------------- | ------------------ | ------------------ | ------------------ | ------- |
+| Northern       | LEKEM              | GONEB              | LANOS              | 359     |
+| Eastern        | VEMDA              | KADRU              | TAVET              | 090     |
+| Southern       | SABAX              | LUTUK              | NOLOX              | 180     |
+| Western        | DUGAD              | LERKO              | DUNNE              | 270     |
+
+- **Vertical Dimensions**:  
+  Lanes extend from `SFC` or the base of restricted airspace to `F160`.
+
+#### Example Departure Clearance via Lane
+
+Aircraft departing ESL military airspace may be instructed to track via a lane at or below `F160`.
+
+!!! example
+    **BEARCAT21** is departing East Sale via the Eastern Lane at FL110. The departure clearance would be issued as follows:
+    <span class="hotline">**ES SMC** -> **BEARCAT21**</span>: "CLEARED TO VEMDA, VIA EASTERN LANE, FPR, FL110, SQUAWK 1234, DEPARTURES 123.3"
```

Comment:
Just a couple of small changes here:

You can remove the <span>, that particular one is used to denote coord rather than ground-air comms

I'd just like to change the *CLEARED TO VEMDA*, to *CLEARED TO YMES*. While it may be what they do IRL, for VATSIM purposes, we don't want to be teaching people to inadvertently impose clearance limits. Best to keep things as standard as possible across Military and Civil airspace

And the ICAO for the callsign BEARCAT is **BRCT**

Reply:
did something get stripped in your message btw? Maybe some sort of HTML?

Reply:
I'm going to guess its the span for the "hotline" that you wanted removed, so removed that.